apps/frontend/src/components/review/ReviewSummary.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useContext, useEffect, useState } from 'react';
+
+import { NavigButton } from 'components/common/NavigButton';
+import UOLoader from 'components/common/UOLoader';
+import NavigationFragment from 'components/questionary/NavigationFragment';
+import {
+  createMissingContextErrorMessage,
+  QuestionaryContext,
+} from 'components/questionary/QuestionaryContext';
+import { UserContext } from 'context/UserContextProvider';
+import { ReviewStatus, UserRole } from 'generated/sdk';
+import { useCheckAccess } from 'hooks/common/useCheckAccess';
+import { isCallEnded } from 'utils/helperFunctions';
+import useDataApiWithFeedback from 'utils/useDataApiWithFeedback';
+import withConfirm, { WithConfirmType } from 'utils/withConfirm';
+
+import { ReviewContextType } from './ReviewQuestionary';
+import ReviewQuestionaryReview from './ReviewQuestionaryReview';
+
+type ReviewSummaryProps = {
+  confirm: WithConfirmType;
+};
+
+function ReviewReview({ confirm }: ReviewSummaryProps) {
```

Comment:
The name of this function (ReviewReview) is a bit weird. Can you update to something more meaningful and understandable what it is about

Reply:
Renamed to ReviewSummary